packages/amplication-client/src/Resource/code-view/FileExplorerNode.tsx

```diff
@@ -42,7 +53,13 @@ export const FileExplorerNode = ({ file, onSelect }: FileExplorerNodeProps) => {
     >
       {file.children?.map((child) => {
         return (
-          <FileExplorerNode file={child} key={child.path} onSelect={onSelect} />
+          <FileExplorerNode
+            file={child}
+            key={child.path}
+            onSelect={onSelect}
+            currentFolder={currentFolder}
+            isLoading={isLoading}
```

Comment:
why do we need to padd isLoading to Children? if the current node is loading we should not have any children, and if we have children, we probably not loading

Reply:
If the logic in this is wrong, can you please suggest the possible changes that I've
to make to get it working properly